server/api/authentication/server.go

```diff
@@ -0,0 +1,28 @@
+package authentication
+
+import (
+	"github.com/tobocop/go-teleport-directory-browser/api/session"
+	"github.com/tobocop/go-teleport-directory-browser/api/user"
+)
+
+type Server struct {
+	Authenticator Authenticator
+	SessionManager session.Manager
+}
+
+func NewServer() Server {
+	userStore := &user.StaticUserStore{
+		Users: map[string]string{
+			// password: password
```

Comment:
```suggestion
			// user: password
```

Reply:
Ahh, I see what happened here. I was trying to let y'all know the password was the string "password", not commenting on the structure of the map. I'll make it clearer